app/models/guide.rb

```diff
@@ -7,9 +7,20 @@ class Guide < ActiveRecord::Base
 
   has_many :editions, dependent: :destroy
 
-  scope :by_author, ->(author_id) { where(editions: { author_id: author_id }) if author_id.present? }
-  scope :in_state, ->(state) { where(editions: { state: state }) if state.present? }
```

Comment:
Looks like the original would have errored unless the join was added elsewhere..